src/main/resources/templates/newTable.html

```diff
@@ -1,23 +1,63 @@
 <!DOCTYPE html>
 <html lang="en" xmlns:th="http://www.thymeleaf.org">
-
 <head>
     <meta charset="UTF-8">
     <title>Title</title>
+    <style>
+      .slot {
+        border: 1px solid black;
+        padding: 10px;
+        margin: 10px;
+      }
+    </style>
 </head>
 <body>
 <div>
     <h1>Dodatu</h1>
-    <form action="/newTable" method="post">
+    <form action="/new-table" method="post">
         <input type="text" name="title" placeholder="nazva">
         <input type="text" name="text" placeholder="text">
         <input type="text" name="location" placeholder="location">
         <input type="number" name="count" placeholder="count">
+        <div id="slot-container">
+            <label for="slot-container">Slots:</label>
+            <div class="slot">
+                <input type="text" name="name" placeholder="name" required>
+                <input type="number" name="costMember" placeholder="costMember" required>
+                <button type="button" onclick="this.parentNode.remove()">Видалити</button>
+            </div>
+        </div>
+        <br /><br />
+        <button type="button" onclick="addSlot()">Додати слот</button>
+        <br /><br />
         <button type="submit">Створити</button>
     </form>
-    <br>
-    <h1>Слоти</h1>
-
 </div>
+<script type="text/javascript">
+      function addSlot() {
```

Comment:
You mentioned that you are going to use some JS framework/lib, not just pure JS for frontend

Reply:
> You mentioned that you are going to use some JS framework/lib, not just pure JS for frontend

Yes,but i will do it latter,when writte more back-end probably